test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs

```diff
@@ -940,6 +951,8 @@ public override void ConfigureWebHost(IServiceCollection services)
                 // The legacy http tests use sync IO so explicitly allow this
                 var environment = new TestEnvironment();
                 environment.SetEnvironmentVariable(EnvironmentSettingNames.AzureWebJobsFeatureFlags, ScriptConstants.FeatureFlagAllowSynchronousIO);
+                environment.SetEnvironmentVariable(EnvironmentSettingNames.MachineIdentifier, "SomeRandomId");
```

Comment:
nit: use realistic values like e777fde04dea4eb931d5e5f06e65b4fdf5b375aed60af41dd7b491cf5792e01b, 89.0.7.73, RD281878FCB8E7 for these?